gorgone/contrib/gorgone_install_plugins.pl

```diff
@@ -44,6 +44,10 @@
 my $command;
 if ($type eq 'rpm') {
     $command = 'yum -y install';
+    `dnf --help`;
```

Comment:
I think you can send the output to /dev/null to avoid the stdout pollution.

Or better, perl proposes things like bash, you should be able to write
```
if (-x '/usr/bin/dnf') { ... }
```
or an equivalent...

Reply:
I have checked, and things prove I'm right:
```
my $command;
if (-x '/usr/bin/dnf') {
  $command = 'dnf -y install';
} else {
  $command = 'yum -y install';
}
```
works perfectly...